src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -386,6 +368,13 @@ public SnowflakeStreamingIngestChannelInternal<?> openChannel(OpenChannelRequest
       // Add channel to the channel cache
       this.channelCache.addChannel(channel);
 
+      // Add storage to the storage manager, only for external volume
+      this.storageManager.addStorage(
```

Comment:
should this call be added when we actually add an external volume manager to main?

Reply:
Make sense, deleted.